tracer/src/Datadog.Trace.ClrProfiler.Native/dd_profiler_constants.h

```diff
@@ -93,6 +93,7 @@ const shared::WSTRING windows_nativemethods_type = WStr("Datadog.Trace.ClrProfil
 const shared::WSTRING appsec_nonwindows_nativemethods_type = WStr("Datadog.Trace.AppSec.Waf.NativeBindings.NativeLibrary+NonWindows");
 const shared::WSTRING appsec_windows_nativemethods_type = WStr("Datadog.Trace.AppSec.Waf.NativeBindings.NativeLibrary+Windows");
 const shared::WSTRING profiler_nativemethods_type = WStr("Datadog.Trace.ContinuousProfiler.NativeInterop+NativeMethods");
+const shared::WSTRING native_loader_nativemethods_type = WStr("Datadog.Trace.Util.RuntimeId+NativeMethods");
```

Comment:
I wonder, should we move the native methods out of `Datadog.Trace.Util.RuntimeId`, into a more general class, e.g. `Datadog.Trace.NativeLoaderMethods`? Feels likely that we'll want to add additional methods in the future, and mean we wouldn't have to change this again?